peps/pep-0702.rst

```diff
@@ -132,16 +132,20 @@ For deprecated classes and functions, this includes:
 * If ``import *`` is used, usage of deprecated objects from the
   module (``from module import *; x = deprecated_object()``)
 * ``from`` imports (``from module import deprecated_object``)
-
-There are some additional scenarios where deprecations could come into play:
-
-* An object implements a :class:`typing.Protocol`, but one of the methods
-  required for protocol compliance is deprecated.
-* A class uses the ``@override`` decorator from :pep:`698` to assert that
-  its method overrides a base class method, but the base class method is
-  deprecated.
-
-As these scenarios appear complex and relatively unlikely to come up in practice,
+* Any syntax that indirectly triggers a call to the function. For example,
+  if the ``__add__`` method of a class ``C`` is deprecated, then
+  the code ``C() + C()`` should trigger a diagnostic. Similarly, if the
+  setter of a property is marked deprecated, attempts to set the property
+  should trigger a diagnostic.
+
+If a method is marked with the :func:`typing.override` decorator from :pep:`698`
+and the base class method it overrides is deprecated, the type checker should
+produce a diagnostic.
+
+There are additional scenarios where deprecations could come into play.
+For example, an object may implement a :class:`typing.Protocol`, but one
+of the methods required for protocol compliance is deprecated.
+As scenarios such as this one appear complex and relatively unlikely to come up in practice,
 this PEP does not mandate that type checkers detect them.
 
 Example
```

Comment:
The example still uses:
```
from typing import deprecated
```
instead of the new location.

Reply:
Thanks, fixed